cloud/aws/lambda/variables.tf

```diff
@@ -0,0 +1,359 @@
+  type        = list
+  default     = []
+}
+
+variable "filter_custom_includes" {
+  description = "List of tags to include when custom filtering is used"
+  type        = list
+  default     = []
+}
+
+variable "filter_custom_excludes" {
+  description = "List of tags to exclude when custom filtering is used"
+  type        = list
+  default     = []
+}
+
+variable "detectors_disabled" {
+  description = "Disable all detectors in this module"
+  type        = bool
+  default     = false
+}
+
+# AWS Lambda detectors specific
+
+# Pct_errors detectors
+
+variable "pct_errors_disabled" {
+  description = "Disable all alerting rules for pct_errors detector"
+  type        = bool
+  default     = null
+}
+
+variable "pct_errors_disabled_critical" {
+  description = "Disable critical alerting rule for pct_errors detector"
+  type        = bool
+  default     = null
+}
+
+variable "pct_errors_disabled_warning" {
+  description = "Disable warning alerting rule for pct_errors detector"
+  type        = bool
+  default     = null
+}
+
+variable "pct_errors_notifications" {
+  description = "Notification recipients list for every alerting rules of pct_errors detector"
+  type        = list
+  default     = []
+}
+
+variable "pct_errors_notifications_warning" {
+  description = "Notification recipients list for warning alerting rule of pct_errors detector"
+  type        = list
+  default     = []
+}
+
+variable "pct_errors_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of pct_errors detector"
+  type        = list
+  default     = []
+}
+
+variable "pct_errors_aggregation_function" {
+  description = "Aggregation function and group by for pct_errors detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".sum(by=['FunctionName'])"
+}
+
+variable "pct_errors_transformation_function" {
+  description = "Transformation function for pct_errors detector (mean, min, max)"
+  type        = string
+  default     = "sum"
+}
+
+variable "pct_errors_transformation_window" {
+  description = "Transformation window for pct_errors detector (i.e. 5m, 20m, 1h, 1d)"
+  type        = string
+  default     = "1h"
+}
+
+variable "pct_errors_threshold_critical" {
+  description = "Critical threshold for pct_errors detector"
+  type        = number
+  default     = 30
+}
+
+variable "pct_errors_threshold_warning" {
+  description = "Warning threshold for pct_errors detector"
+  type        = number
+  default     = 20
+}
+
+variable "pct_errors_aperiodic_duration" {
+  description = "Duration for the pct_errors block"
+  type        = string
+  default     = "10m"
+}
+
+variable "pct_errors_aperiodic_percentage" {
+  description = "Percentage for the pct_errors block"
+  type        = number
+  default     = 0.9
+}
+
+# Errors detectors
+
+variable "errors_disabled" {
+  description = "Disable all alerting rules for errors detector"
+  type        = bool
+  default     = null
```

Comment:
should be true

Reply:
We want error detectors to all be disabled by default?

Reply:
We have been setting detectors_disabled to false in all modules and the the disabled variables for specific detectors to null

Reply:
see my answer for sqs

Reply:
done